core/App/ehsm_napi.cpp

```diff
@@ -1380,13 +1389,15 @@ char *NAPI_RA_GET_API_KEY(const char *p_att_result_msg)
     retJsonObj.addData_uint8Array("nonce", pt_att_result_msg->platform_info_blob.nonce.rand, 16);
     retJsonObj.addData_string("appid", p_appid);
     retJsonObj.addData_string("apikey", (char*)p_apikey.data);
+    retJsonObj.addData_uint8Array("cipherapikey", cipherapikey.data, cipherapikey.datalen);
 
     log_d("apikey_result_msg: \n%s",retJsonObj.toChar());
     log_d("***NAPI_RA_GET_API_KEY end.");
 
 OUT:
     explicit_bzero(p_apikey.data, p_apikey.datalen);
     SAFE_FREE(pt_att_result_msg);
+    SAFE_FREE(cipherapikey.data);
     SAFE_FREE(p_apikey.data);
```

Comment:
does the duplicate for the line#1388 and line#1390?

Reply:
seems it is